pom.xml

```diff
@@ -111,6 +139,11 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
+            <version>3.12.4</version>
+            <!-- Note: mockito 4.x removed the Matchers class,
+                 which was Deprecated in 3.x and is currently
+                 used in MatrixNotificationTest sources
+            -->
```

Comment:
`org.mockito.Matchers` was deprecated in favor of `org.mockito.ArgumentMatchers`. Better to adapt to the new name rather than to continue using a deprecated version.

Reply:
Seems to build, thanks :)